### ISSUE TITLE
npm install failed

### ISSUE THREAD

**Issue by sce9sc** [hampoelz/Capacitor-NodeJS#11]
I tried to install capacitor-nodejs on a clean project and I got the bellow results 

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: capacitor-app@1.0.0
npm ERR! Found: @capacitor/core@5.0.5
npm ERR! node_modules/@capacitor/core
npm ERR!   peer @capacitor/core@"^5.0.0" from @capacitor/android@5.0.5
npm ERR!   node_modules/@capacitor/android
npm ERR!     @capacitor/android@"^5.0.5" from the root project
npm ERR!   peer @capacitor/core@"^5.0.0" from @capacitor/camera@5.0.4
npm ERR!   node_modules/@capacitor/camera
npm ERR!     @capacitor/camera@"latest" from the root project
npm ERR!   2 more (@capacitor/splash-screen, the root project)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! capacitor-nodejs@"https://github.com/hampoelz/capacitor-nodejs/releases/download/v1.0.0-beta.2/capacitor-nodejs.tgz" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @capacitor/core@4.8.0
npm ERR! node_modules/@capacitor/core
npm ERR!   peer @capacitor/core@"^3.0.0 || ^4.0.0" from capacitor-nodejs@1.0.0-beta.2
npm ERR!   node_modules/capacitor-nodejs
npm ERR!     capacitor-nodejs@"https://github.com/hampoelz/capacitor-nodejs/releases/download/v1.0.0-beta.2/capacitor-nodejs.tgz" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/sce9sc/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sce9sc/.npm/_logs/2023-06-22T20_08_23_735Z-debug-0.log

```

my package json is 

```
{
  "name": "capacitor-app",
  "version": "1.0.0",
  "description": "An Amazing Capacitor App",
  "main": "index.js",
  "keywords": [
    "capacitor",
    "mobile"
  ],
  "scripts": {
    "start": "vite",
    "build": "vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "@capacitor/android": "^5.0.5",
    "@capacitor/camera": "latest",
    "@capacitor/core": "latest",
    "@capacitor/splash-screen": "latest"
  },
  "devDependencies": {
    "@capacitor/cli": "latest",
    "vite": "^2.9.13"
  },
  "author": "",
  "license": "ISC"
}
```

Thank you.

**Comment by hampoelz** [comment]
Thanks for your report. This issue occurs because I haven't updated the plugin to Capacitor v5 yet. As a workaround you can use `npm install --force` for now. Everything should still work without problems.

Next week I'm on vacation, after that I plan to update this project and work more actively on it again.

**Comment by hampoelz** [comment]
Fixed in v1.0.0-beta.3
